core/modules/file/file.install

```diff
@@ -695,9 +740,106 @@ function file_update_1001() {
 }
 
 /**
- * Switch the 'edit files' permission from file_entity to 'manage files'.
+ * Add {file_managed}.type field
  */
 function file_update_1002() {
```

Comment:
Looks like this needs to be bumped into `file_update_1003()`, incorrect ordering probably caused by `git merge`.